apps/total-typescript/src/pages/api/subscriber.ts

```diff
@@ -1,5 +1,52 @@
-import convertkitSubscriberHandler from '@skillrecordings/convertkit/dist/api/subscriber'
 import {withSentry} from '@sentry/nextjs'
+import {NextApiRequest, NextApiResponse} from 'next'
+import {prisma, User} from '@skillrecordings/database'
+import {getSubscriberByEmail} from 'lib/get-subscriber-by-email'
+import {fetchSubscriber} from '@skillrecordings/convertkit-sdk'
+import {CK_SUBSCRIBER_KEY} from '@skillrecordings/config'
+import {serialize} from 'cookie'
+
+const convertkitSubscriberHandler = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+) => {
+  if (req.method === 'GET') {
+    let subscriber
+
+    if (req.cookies['ck_subscriber']) {
+      subscriber = JSON.parse(req.cookies['ck_subscriber'])
+    } else if (req.query.learner) {
```

Comment:
if they visit with `learner` query param we look the id up to see if we've got a match